custom/ilsgateway/tanzania/reports/stock_on_hand.py

```diff
@@ -123,23 +121,23 @@ def get_previous_month(self, enddate):
         return month, year
 
     def get_stockouts(self, facs):
-        facs_count = facs.count()
-        if facs_count > 0:
-            fac_ids = facs.exclude(supply_point_id__isnull=True).values_list('supply_point_id', flat=True)
-            enddate = self.config['enddate']
-            month, year = self.get_previous_month(enddate)
-            stockouts = StockTransaction.objects.filter(
-                case_id__in=list(fac_ids),
-                stock_on_hand__lte=0,
-                report__domain=self.config['domain'],
-                report__date__range=[
-                    datetime(year, month, 1),
-                    datetime(enddate.year, enddate.month, 1)
-                ]
-            ).order_by('case_id').distinct('case_id').count()
-            percent_stockouts = (stockouts or 0) * 100 / float(facs_count)
-        else:
-            percent_stockouts = 0
+        if not facs:
```

Comment:
Sorry, perhaps I misread this the first time but I see you are doing a 'count' later on as well so it seems fine to do it here first and then use that in the if.